calcom/cal.com
# fix: broken url on booking page

## What does this PR do?



- Fixes https://github.com/calcom/cal.com/issues/15021
- Fixes CAL-3702 (Linear issue number - should be visible at the bottom of the GitHub issue description)

Regression from https://github.com/calcom/cal.com/pull/14777/files.



## Mandatory Tasks (DO NOT REMOVE)


## How should this be tested?

1) Go to any team event type url and make sure team logo is not visible. (for non org event types)
